replayserver/streams/delayed.py

```diff
@@ -66,17 +66,41 @@ def _data_length(self):
         return min(len(self._stream.data), self._current_position)
 
     def _data_slice(self, s):
```

Comment:
without comment, typing and without proper naming this code is available only for one person.

Reply:
interface documentation is in replayserver/streams/base.py